BismarkUnchained/scenes/Bullet/Body.gd

```diff
@@ -7,9 +7,9 @@ func _physics_process(delta):
 	rotation = velocity.angle()
 	
 	# move and check for a collision
-	var kc = move_and_collide(velocity * delta)
+	var kc = move_and_collide(velocity * get_node("/root/Main").time_delta * delta)
```

Comment:
idk how strict we want to be about using getter/setter. setter is more important than getter for sure, but I think we should be using both